src/main/java/org/example/Main.java

```diff
@@ -4,7 +4,7 @@
 
 public class Main {
     public static void main(String[] args) {
-        List<String> drinkList = List.of("Tea", "Coffee", "Orange juice", "Apple juice", "Green tea");
+        List<String> drinkList = List.of("Tea", "Coffee", "Orange juice", "Apple juice", "Green tea", "Cocoa");
```

Comment:
対応ありがとうございます！
では、ためしに、
```suggestion
        List<String> drinkList = List.of("Tea", "Coffee", "Orange juice", "Apple juice", "Green tea", "Cocoa", "Oolong tea");
```
というようにしてみてほしいです！

Reply:
ご確認いただきありがとうございます。
対応いたしました。
dd1dd4d71f459e96361e0e3198e6f1f2802e21b0